src/flows/components/panel/PanelQueryOverlay.tsx

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import React, {
+  useContext,
+  FC,
+  memo,
```

Comment:
What caused the addition of the memo? That thing haunted my earlier code, so I'm sensitive about it, but lemme know if it was useful for something specific

Reply:
Copy Pasta. Removed.